### ISSUE TITLE
error in running demo

### ISSUE THREAD

**Issue by sjsy** [phoenix104104/LapSRN#1]
Hi,
System configuration:
- Windows 10
- Matlab 2017a
- Visual Studio 2013

\>> demo_LapSRN
Warning: Name is nonexistent or not a directory:
D:\Users\user\sr\LapSNR\LapSRN-master\LapSRN-master\matconvnet\matlab 
\> In path (line 109)
  In addpath (line 88)
  In demo_LapSRN (line 25) 
Load pretrained_models\LapSRN_x4.mat
Undefined function or variable 'dagnn.vllab_dag_loss'.

Error in dagnn.DagNN.loadobj (line 26)
    block = constr() ;

Error in demo_LapSRN (line 33)
net = dagnn.DagNN.loadobj(net.net);

Any help please?
Thanks.

**Comment by sjsy** [comment]
The dagnn.vllab_dag_loss had not been copied by install.m
I copied it manually!
Best.

**Comment by trillionpowers** [comment]
My system is ubuntu 14.04. The environment is matlab R2015b + CUDA8.0.
My matconvnet is installed in this path /usr/local/MATLAB/R2015b/toolbox/Matconvnet/matconvnet-1.0-beta24. I have tried install.m and is successful.###  I also tried copy "vllab_dag_loss.m" to matconvnet/matlab/+dagnn. But it just can not found this object. The errors are like this:

demo_LapSRN
Warning: Name is nonexistent or not a directory:
/media/trillionpowers/Data/Work/RL_SR/Basical_Model/LapSRN_model/LapSRN-master/usr/local/MATLAB/R2015b/toolbox/Matconvnet/matconvnet-1.0-beta24/matlab
In path (line 109)
In addpath (line 88)
In demo_LapSRN (line 25)
Load pretrained_models/LapSRN_x4.mat
Error using dagnn.DagNN.loadobj
Error: File: loadobj.m Line: 26 Column: 5
The input character is not valid in MATLAB statements or expressions.
Error in demo_LapSRN (line 33)
net = dagnn.DagNN.loadobj(net.net);


 I have copied this file to this path many times and use 'vl_compilenn('enableGPu', true);' to compile.
But I still have this error when I run 'demo_LapSRN.m' and 'train_LapSRN.m'.
I have tried to Change permissions of this file in 777. But it still can not run.

Thank you for your help @sjsy
